sphinx_rtd_theme/layout.html

```diff
@@ -228,6 +228,9 @@
     })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
 
     ga('create', '{{ theme_analytics_id }}', 'auto');
+    {% if theme_analythics_anonymize_ip %}
```

Comment:
```suggestion
    {% if theme_analytics_anonymize_ip|tobool %}
```

Reply:
Thanks for spotting the typo and I did not know about the tobool filter